Client.Tests/ModifyTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using FakeItEasy;
+using FluentFTP;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using static Client.Program;
+
+namespace Client.Tests
+{
+    public class ModifyTests
+    {
+        FtpClient testClient = A.Fake<FtpClient>();
+
+        [Fact]
+        public void Rename_LocalFlagSuccess()
+        {
+            Directory.CreateDirectory($@"{Directory.GetCurrentDirectory}UnitTesting");
+
+            Directory.CreateDirectory(($@"{Directory.GetCurrentDirectory}UnitTesting\TestFile.txt"));
```

Comment:
This isn't actually creating a text file, just a directory called `TestFile.txt`. Also, `{Directory.GetDirectory()}` can be removed from all these tests.

Reply:
Or we can just get rid of it entirely